source/install.txt

```diff
@@ -10,168 +10,242 @@ Install Compass
    :depth: 1
    :class: singlecol
 
-Compatibility
+Prerequisites
 -------------
 
+Software Requirements
+~~~~~~~~~~~~~~~~~~~~~
+
 |compass| works with MongoDB Server version 3.0 and higher, both
 Enterprise and Community editions. It runs on 64-bit versions
-of Mac OS X, Windows, Ubuntu 14.04+ and RHEL 7+.
+of Mac macOS, Windows, Ubuntu 14.04+ and RHEL 7+.
 
 Software requirements to run |compass-short| are listed below:
 
 .. include:: /includes/fact-compass-platforms.rst
 
-Installation
-------------
-
 .. note::
 
-   As of MongoDB Server version 3.6, MongoDB packages a
-   platform-specific |compass-short| installation script with
-   `MongoDB Server downloads
-   <https://www.mongodb.com/download-center/enterprise>`_. For details,
-   see the :manual:`install_compass script
-   </reference/program/install_compass/>` documentation.
+   The Windows installation of |compass| requires
+   **Microsoft .NET Framework version 4.5 or later.** The
+   |compass-short| installer prompts you to install the
+   minimum required version of the .NET framework if it is
+   not already installed on your system.
+
+Network Requirements
+~~~~~~~~~~~~~~~~~~~~
+
+Certain features of |compass|, such as third-party mapping services and
+Intercom, require inbound connections from external sources to
+function. In some instances, users have reported issues with their
+system firewall blocking these network requests and thereby limiting
+|compass| functionalities. To fix this issue, configure your system
+firewall to allow incoming connections for |compass|.
+
+.. important::
```

Comment:
I'm wondering if this needs to be in the important block since it's more of an instruction relating to the previous sentence. Thoughts?

Reply:
No strong opinion here - I appreciate that it's called out in a note, but understand your point. @jeff-allen-mongo